hebrew/bar/bar.md

```diff
@@ -7,4 +7,6 @@
 
 עישנת לעצמך איזה ג'וינט טוב.
 
-מה עכשיו?
\ No newline at end of file
+מה עכשיו?
+[תניח תפילין בארץ הקודש](HolyLand/holyLand.md)
+[תקטוף תותים מדבר](Strawberries/desert-strawberries.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.